kokoro/macos/build.sh

```diff
@@ -58,9 +58,7 @@ function build {
 
 # Build each API package separately first, as the go-compiler needs ~8GB of
 # RAM for each of the big API packages.
-for api in gles vulkan gvr; do
-  build //gapis/api/$api:go_default_library
-done
+build //gapis/api/vulkan:go_default_library
```

Comment:
We probably want to build everything BUT vulkan first, then vulkan.
The old code built gles (and all dependencies) then started on the rest. This means that vulkan would build by itself, now it will build with all its dependencies, might cause some issues?
But also might not.

Reply:
No, this is fine. This is no different than before, where the deps were built as part of the first API. What kills the VM is building the API package, which is always going to be the last thing built and on its own, as it is the only root of the dependency tree.